darts/tests/models/forecasting/test_regression_models.py

```diff
@@ -884,14 +865,44 @@ def test_static_covs_addition(self):
         self.assertEqual(features.shape, (expected_height, expected_width))
 
         # outptut_chunk_length < max_future_cov_lag and len(target_series) = len(future_covs)
+        target_series = [series1, series2, series3]
+        past_covs = None
+        future_covs = [series1, series2, series3]
         target_lag = [-1]
         past_covs_lag = None
-        future_covs_lag = [-1, 3]
+        future_covs_lag = [-1, -3]
```

Comment:
here it was a case to test when the output_chunk_length is smaller than the positive lag for future covariates (maybe it is not a case that happens often in practice) but in the previous implementation of _create_lagged_data it was a case that caused errors in the back computation of the length and width of the features. Would this still be relevant to test here?